apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js

```diff
@@ -72,9 +72,10 @@ const copyStylesToIframe = ( srcDocument, targetiFrameDocument ) => {
  * @param {object} props.className CSS class to apply to component
  * @param {string} props.bodyClassName CSS class to apply to the iframe's `<body>` tag
  * @param {number} props.viewportWidth pixel width of the viewable size of the preview
- * @param {Array}  props.blocks array of Gutenberg Block objects
+ * @param {Array} props.blocks array of Gutenberg Block objects
  * @param {object} props.settings block Editor settings object
  * @param {Function} props.setTimeout safe version of window.setTimeout via `withSafeTimeout`
+ * @param props.title Template Title - see #39831 for details.
```

Comment:
`yarn run dev` in FSE automatically added this param, I needed to add a description to get it to pass the commit hook.

Reply:
Does that need a `{string}` prepended?

Reply:
Fixed in 5e8ef98e6